backend/src/main/resources/graphql/main.graphqls

```diff
@@ -262,6 +264,8 @@ type TestOrder {
   dateTested: DateTime
   correctionStatus: String
   reasonForCorrection: String
+  createdBy: ApiUser
+  patientLink: PatientLink
```

Comment:
whats the purpose of adding extra fields to the TestOrder?

Reply:
Ain't none, removed in https://github.com/CDCgov/prime-simplereport/pull/6682/commits/34346c5a58c35cca7a4fde47695fcf9e994ce8ae.